jta-1_2-standalone/pom.xml

```diff
@@ -158,15 +122,18 @@
                     <target>${maven.compiler.target}</target>
                 </configuration>
             </plugin>
-
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-surefire-plugin</artifactId>
                 <version>${version.maven-surefire-plugin}</version>
                 <configuration>
                     <systemPropertyVariables>
-                        <com.arjuna.ats.arjuna.objectstore.objectStoreDir>${project.build.directory}/defaultstore</com.arjuna.ats.arjuna.objectstore.objectStoreDir>
-                        <com.arjuna.ats.arjuna.common.ObjectStoreEnvironmentBean.objectStoreDir>${project.build.directory}/commstore</com.arjuna.ats.arjuna.common.ObjectStoreEnvironmentBean.objectStoreDir>
+                        <com.arjuna.ats.arjuna.objectstore.objectStoreDir>
+                            ${project.build.directory}/tx-object-store
+                        </com.arjuna.ats.arjuna.objectstore.objectStoreDir>
+                        <com.arjuna.ats.arjuna.common.ObjectStoreEnvironmentBean.objectStoreDir>
+                            ${project.build.directory}/tx-object-store
+                        </com.arjuna.ats.arjuna.common.ObjectStoreEnvironmentBean.objectStoreDir>
```

Comment:
Setting the property by two different mechanisms might confuse the quickstart user